app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/address/PropertyDetailsController.scala

```diff
@@ -945,7 +946,7 @@ class PropertyDetailsController @Inject() (
   protected lazy val enterNonUkAddressCall: Call                          = enterPostcodeCall
   protected lazy val enterNonUkAddressSubmitCall: Call                    = enterPostcodeCall
   protected lazy val backLinkCall: FillingOutReturnAddressJourney => Call = _ => enterPostcodeCall
-
+  protected lazy val exitPageCall: Call                                   = routes.PropertyDetailsController.showExitPage()
```

Comment:
wasn't sure about this - we don't really need the exit page for this part of the service functionality

Reply:
maybe a way to avoid having to do this where we don't need it is to have the type of this `Option[Call]` and default it to `None`, then override it where we do need it. Then we could avoid having to figure out whether or not it's register with or without id in the `AddressController` since you'd just do something like this if the person chose yes to the is uk question:
```scala
 Redirect(ukAddressNotAllowedExitPageCall.getOrElse(enterPostcodeCall)
```
I put a more descriptive name in there as well.

Reply:
changed to make the call optional